apps/judicial-system/web/src/components/Table/PastCasesTable/PastCasesTable.tsx

```diff
@@ -0,0 +1,161 @@
+import React, { useContext, useMemo } from 'react'
+import cn from 'classnames'
+import { theme } from '@island.is/island-ui/theme'
+import { Box, Text } from '@island.is/island-ui/core'
+
+import { useIntl } from 'react-intl'
+import { capitalize } from '@island.is/judicial-system/formatters'
+import { tables, core } from '@island.is/judicial-system-web/messages'
+import {
+  CaseListEntry,
+  isExtendedCourtRole,
+} from '@island.is/judicial-system/types'
+
+import {
+  useSortCases,
+  useViewport,
+} from '@island.is/judicial-system-web/src/utils/hooks'
+
+import {
+  UserContext,
+  TagAppealState,
+  TagCaseState,
+} from '@island.is/judicial-system-web/src/components'
+
+import {
+  ColumnCaseType,
+  CourtCaseNumber,
+  DefendantInfo,
+  SortButton,
+  TableContainer,
+  TableHeaderText,
+  DurationDate,
+  getDurationDate,
+} from '@island.is/judicial-system-web/src/components/Table'
+
+import MobilePastCase from './MobilePastCase'
+import * as styles from '../Table.css'
+
+interface Props {
+  cases: CaseListEntry[]
+  onRowClick: (id: string) => void
+  loading?: boolean
+  testid?: string
+}
+
+const PastCasesTable: React.FC<Props> = (props) => {
+  const { cases, onRowClick, loading = false, testid } = props
+  const { formatMessage } = useIntl()
+  const { user } = useContext(UserContext)
+
+  const { sortedData, requestSort, getClassNamesFor } = useSortCases(
+    'createdAt',
+    'descending',
+    cases,
+  )
+  const pastCasesData = useMemo(
+    () =>
+      cases.sort((a: CaseListEntry, b: CaseListEntry) =>
+        a['created'].localeCompare(b['created']),
+      ),
+    [cases],
+  )
+
+  const { width } = useViewport()
+
+  return width < theme.breakpoints.md ? (
+    <>
+      {pastCasesData.map((theCase) => (
+        <Box marginTop={2} key={theCase.id}>
+          <MobilePastCase
+            theCase={theCase}
+            onClick={() => onRowClick(theCase.id)}
+            isCourtRole={false}
+          >
+            <DurationDate
+              key={`${theCase.id}-duration-date`}
+              date={getDurationDate(
+                theCase.state,
+                theCase.validToDate,
+                theCase.initialRulingDate,
+                theCase.courtEndTime,
+              )}
+            />
+          </MobilePastCase>
+        </Box>
+      ))}
+    </>
+  ) : (
+    <TableContainer
+      loading={loading}
+      testid={testid}
+      tableHeader={
+        <>
+          <TableHeaderText title={formatMessage(tables.caseNumber)} />
+          <th className={cn(styles.th, styles.largeColumn)}>
+            <SortButton
+              title={capitalize(formatMessage(core.defendant, { suffix: 'i' }))}
+              onClick={() => requestSort('defendant')}
+              sortAsc={getClassNamesFor('defendant') === 'ascending'}
+              sortDes={getClassNamesFor('defendant') === 'descending'}
+            />
+          </th>
+          <TableHeaderText title={formatMessage(tables.type)} />
```

Comment:
In the current past cases table we can sort cases by type, after this we'll only be able to sort by defendant. I'm actually not sure which is right though as the active cases table can't be sorted by type. Should we check with Guðný if it's ok that we remove this sorting, or did you already talk to her?
It does feel like a weird sorting by type to me since we also have the dropdown filter at the top, but there might be a reason for it being sortable as well that I'm not aware of